tests/clock_mux/README.rst

```diff
@@ -0,0 +1,29 @@
+Clock multiplexing primitive
+++++++++++++++++++++++++++++
+
+This is an example of modeling a clock mux/buffer which has both clock inputs and clock outputs.
+
+According to the "Primitive Block Timing Modeling Tutorial" section "Clock Buffers & Muxes" of the `VTR documentation <https://docs.verilogtorouting.org/en/latest/tutorials/arch/timing_modeling/#clock-muxes>`_, a clock buffer or a clock mux has to have its outputs(s) as combinational sinks of clock input(s). This contradicts with how regular sequential blocks (like flip-flops) have to be modeled.
+
+V2X provides the attribute `(* NO_COMB *)` that when specified on a clock port and given explicitly the value of 0 will prevent it from getting combinational_sink annotations in XML.
+
```

Comment:
As discussed on VC, the name is too confusing. I think we should rework this to maybe be a `(* COMB_INCLUDE_CLOCKS *)` attribute on the output?

Reply:
That's a good idea. I've re-implemented the functionality so now there is the `(* COMB_INCLUDE_CLOCKS *)` attribute that has to be specified on an output port.